### DISCUSSION TITLE
Getting Error while running test (Cuckoo installed via Swift Package Manager).

### DISCUSSION THREAD

**Discussion by ubitCloud** [Brightify/Cuckoo#504] [Q&A] [answered]
Getting error when running test after installing Cuckoo via SPM. The error is related to the XcodeProj dependency.

Project repo: https://github.com/ubitCloud/CuckooP3

Screenshots attached.
<img width="318" alt="Screenshot 2024-07-22 at 4 27 21 PM" src="https://github.com/user-attachments/assets/a9284994-6bcb-4aca-9e03-91ce2c0ac54e">
<img width="920" alt="Screenshot 2024-07-22 at 5 24 04 PM" src="https://github.com/user-attachments/assets/c8f4132f-0afa-4bcc-b6cb-50ae7c662e31">

Facing this issue: https://github.com/Brightify/Cuckoo/issues/496
XcodeProj issue: https://github.com/tuist/XcodeProj/issues/817

@MatyasKriz 
Thanks. ☺️

**Comment by MatyasKriz** [comment]
That issue is closed because it's already resolved. Try to follow my last comment there carefully. Don't add Cuckoo to the production target, only to the tests.

**Reply by ubitCloud** [reply]
@MatyasKriz still facing same issue, I have update the cuckoofile.toml as you mentioned in the last commit to remove generator.

also I have add both cuckoo and cuckooGenerator to the test target only.

<img width="973" alt="Screenshot 2024-07-23 at 12 18 06 PM" src="https://github.com/user-attachments/assets/f72b6187-0848-47cb-a5c9-ad96d5e931ba">
<img width="619" alt="Screenshot 2024-07-23 at 12 27 01 PM" src="https://github.com/user-attachments/assets/3bf4caf4-e7cb-42ae-850d-647c940c4506">
<img width="487" alt="Screenshot 2024-07-23 at 12 37 42 PM" src="https://github.com/user-attachments/assets/9e5fe933-6e4e-4172-adab-a06d22b5473b">

Project repo: https://github.com/ubitCloud/CuckooP3

**Reply by MatyasKriz** [reply]
As I said in the comment in the issue you linked.. "CuckooGenerator" isn't supposed to be used by you. Only "Cuckoo".

<img width="608" alt="image" src="https://github.com/user-attachments/assets/266229be-24d6-4143-aa97-c28b42b0ac3f">

**Reply by MatyasKriz** [reply, marked as answer]
SPM generator ignores the output parameter, there's an issue for that, but the project tests will still see the generated mocks in derived data.

**Reply by ubitCloud** [reply]
@MatyasKriz successfully generated mocks thanks you so much it means a lot to me. 😊
